### DISCUSSION TITLE
Timer-ID "issue"

### DISCUSSION THREAD

**Discussion by gvanem** [cesanta/mongoose#2584] [Ask questions, get answers]
I was greeping through the sources to figure out what `mg_timer::id` was used for. Searched for `t->id` and found nothing!
But the way some lines are written; like:
```c
      struct mg_connection *t;
      for (t = c->mgr->conns; t != NULL; t = t->next) {
        if (t->id == *id) {
          ....
```

mislead me to believe it was a timer. Not the case. So could you perhaps use `_c` instead:
```c
      struct mg_connection *_c;
      for (_c = c->mgr->conns; _c != NULL; _c = _c->next) {
        if (_c->id == *id) {
          ....
```

**Comment by cpq** [comment]
Heh, yeah... `t` meant "temporary" :)
